README.md

```diff
@@ -9,17 +9,42 @@ A nice swipe layout that provides new actions with a material design look and fe
 <img src="/images/demo1.gif" alt="Demo Screen Capture" width="300px" />
 <img src="/images/demo2.gif" alt="Demo Screen Capture" width="300px" />
 
+## Installation
+
+Step 1. Add the JitPack repository to your build file
+
+```groovy
+allprojects {
+  repositories {
+    ...
+    maven { url 'https://jitpack.io' }
+  }
+}
+```
+
+Step 2. Add the dependency
+
+```groovy
+dependencies {
+  implementation 'com.github.prolificinteractive.patrons:patron:${patronsVersion}'
```

Comment:
so in your case: `implementation 'com.github.prolificinteractive:Chandelier:${chandelierVersion}'`